easybuild/easyconfigs/s/Simple-DFTD3/Simple-DFTD3-0.7.0-intel-2022b.eb

```diff
@@ -0,0 +1,50 @@
+# A. Domingo (Vrije Universiteit Brussel)
+# J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeNinja'
+
+name = 'Simple-DFTD3'
+version = '0.7.0'
+
+homepage = 'https://dftd3.readthedocs.io'
+description = """Reimplementation of the D3 dispersion correction. The s-dftd3 project aims to
+provide a user-friendly and uniform interface to the D3 dispersion model and
+for the calculation of DFT-D3 dispersion corrections."""
+
+toolchain = {'name': 'intel', 'version': '2022b'}
+# In EasyBuild 4.8.2 is a bug in the Intel toolchain, so we need to turn off OpenMP here and add
+# -fopenmp instead as Bart Oldman suggested
+toolchainopts = {'usempi': False, 'openmp': False, 'pic': True, 'extra_fflags': '-fopenmp'}
+
+github_account = 'dftd3'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['19400a176eb4dcee7b89181a5a5f0033fe6b05c52821e54896a98448761d003a']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('Ninja', '1.11.1'),
+]
+
+dependencies = [
+    ('mctc-lib', '0.3.1'),
+    ('mstore', '0.2.0'),
+    # ('TOML-Fortran', '0.4.1'), # Vesion used in code is 0.2.4
+]
+
+build_shared_libs = True
+
+configopts = '-DWITH_BLAS=TRUE -DBLAS_LIBRARIES="-lmkl_sequential -lmkl_intel_lp64 -lmkl_core" -DWITH_OpenMP=TRUE '
```

Comment:
You should be able to always use $LIBBLAS instead of hardconding it. LIBBLAS gets set by the toolchain.
I.e. -DBLAS_LIBRARIES="$LIBBLAS"